### PR TITLE
Adding Psychic

### PR DESCRIPTION
**Pull request by blizma**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

**Comment by lepon01** [comment]
Should you reply with an error?

hmmmm

**Comment by lepon01** [comment]
You seem to have difficulty subtracting numbers in the maths category

hmmmmmmm

File: data/en/bots/495743974028476446.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- [discord.py](https://github.com/Rapptz/discord.py/) ily bb <3
+
+# SERIOUS PROBLEMS
+- THE BOT CAN'T REMEMBER ANYTHING!! (NO DATABASES) THAT MEANS WE CAN'T MAKE COOL COMMANDS LIKE `p!warn <@user>` OR `p!mute <@user> [time] [reason]` ORRRRRR MAKE AN ECONOMY / LEVEL SYSTEM TO HELP MAKE THE BOT MORE APPEALING AND INTERESTING!!!!!
+
+# Introduction.
+- Hey, I am a bot made for handy utility and some nice commands to help your discord server, for example, we have a kick command and a ban command to help moderate your server. And also we have some fun commands to help spice up your server to not be that boring. 
+
+# Commands
+- There is a cool handy dandy [documentation page made on the bot](https://psychic.blizma.us) if you wish to see, woah isn't that *cool*.
```
Comment (lepon01) [inline data/en/bots/495743974028476446.md]: Link is broken

File: data/en/bots/495743974028476446.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+# Support Server
+- You can join the support server by [clicking here](https://discord.gg/zEvdVEh)!
+
+# Current Bot Lists
```
Comment (lepon01) [inline data/en/bots/495743974028476446.md]: Eww

File: data/en/bots/495743974028476446.md (hunk 1 above)
Comment (blizma) [reply data/en/bots/495743974028476446.md]: Fixed.

Comment (lepon01) [reply data/en/bots/495743974028476446.md]: Is the DNS propagating now?
I'll test that again soon.

Comment (blizma) [reply data/en/bots/495743974028476446.md]: Nah, the link was broken, I'm gonna buy a domain (for Psychic and not me) soon but I'm gonna use the default link.

Comment (lepon01) [reply data/en/bots/495743974028476446.md]: If you "fixed the link", how is there no new commit on this pull request?

Comment (lepon01) [reply data/en/bots/495743974028476446.md]: not imgur

**Pull request merged**